buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -637,14 +637,38 @@ def get_date() {
 }
 
 /*
-* Set TESTS_TARGETS, indicating the level of testing
-* and sets TEST_FLAG for all targets if defined in variable file
+* Set TESTS_TARGETS, indicating the level of testing.
 */
 def set_test_targets() {
-    TESTS_TARGETS = params.TESTS_TARGETS
-    if (!TESTS_TARGETS) {
-        // set default TESTS_TARGETS for pipeline job (fetch from variables file)
-        TESTS_TARGETS = get_default_test_targets()
+    TESTS_TARGETS = (params.TESTS_TARGETS) ? params.TESTS_TARGETS.trim() : "none"
+    TARGET_NAMES = []
+
+    if (TESTS_TARGETS != 'none') {
+        for (target in TESTS_TARGETS.replaceAll("\\s","").toLowerCase().tokenize(',')) {
+            switch (target) {
+                case "sanity":
```

Comment:
Please combine the 2 case statements, e.g. `case ['sanity', 'extended']`.

Reply:
done